requirements.txt

```diff
@@ -1,24 +1,29 @@
+atlassian-python-api
 backports.ssl-match-hostname>=3.5.0.1
+cffi
 CouchDB==1.1
 ecdsa>=0.13
 genologics @ git+https://github.com/SciLifeLab/genologics.git
 genologics_sql @ git+https://github.com/SciLifeLab/genologics_sql.git
 httplib2>=0.10.3
 ibmcloudant>=0.9.1
-cffi
-matplotlib>=2.0.0
 markdown
+matplotlib>=2.0.0
 nest_asyncio
+nose
 numpy>=1.12.1
 oauth2>=1.9.0.post1
 openpyxl
-atlassian-python-api
+pandas
+psycopg2
```

Comment:
This will fail on ngi-internal if its not already installed since ngi-internal doesn't have a c compiler installed that can compile psycopg2. The compiled version is installed with conda. Not sure how it works on github actions though.

Reply:
@aanil how do you feel about merging the `conda_environment.yml` and `requirements.txt` file into a single file?

The following was installable on ngi-internal
```
channels:
  - conda-forge
  - defaults
dependencies:
  - python=3.12
  - pango>=1.42.0
  - pandas>=1.3.2
  - psycopg2
  - open-fonts
  - xorg-libxrender
  - xorg-libxext
  - xorg-libxau
  - pip:
    - atlassian-python-api
    - backports.ssl-match-hostname>=3.5.0.1
    - cffi
    - CouchDB==1.1
    - ecdsa>=0.13
    - genologics @ git+https://github.com/SciLifeLab/genologics.git
    - genologics_sql @ git+https://github.com/SciLifeLab/genologics_sql.git
    - httplib2>=0.10.3
    - ibmcloudant>=0.9.1
    - markdown
    - matplotlib>=2.0.0
    - nest_asyncio
    - nose
    - numpy>=1.12.1
    - oauth2>=1.9.0.post1
    - openpyxl
    - pycryptodome>=3.6.1
    - pyparsing>=2.2.0
    - python-dateutil>=2.7.5
    - pytz>=2018.9
    - PyYAML
    - requests>=2.14.2
    - selenium
    - setuptools
    - simplejson>=3.10.0
    - six>=1.10.0
    - slack_sdk
    - tornado>=6.0.2
    - utils>=0.9.0
    - weasyprint==52.5
    - Zenpy

```

Reply:
If not, how's this?
[0c49d0f](https://github.com/NationalGenomicsInfrastructure/genomics-status/pull/882/commits/0c49d0ffffc3ab2dd02386054d0479b496d24fe2)

Reply:
> @aanil how do you feel about merging the `conda_environment.yml` and `requirements.txt` file into a single file?

I'm all for it if it works! What command would you use to install it?

Reply:
While it is possible to list nested pip requirements within a conda .yml file, it's not really compatible with the `setup.py` file which only expects pip packages and is not well adapted to handling conda .env files, so I think the current approach ([0c49d0f](https://github.com/NationalGenomicsInfrastructure/genomics-status/pull/882/commits/0c49d0ffffc3ab2dd02386054d0479b496d24fe2)) might be the most reasonable compromise.